src/jsonschema.rs

```diff
@@ -18,10 +20,15 @@ enum Type {
     Array,
 }
 
+enum Type {
+    Atom(Atom),
+    List(Vec<Atom>),
+}
+
 #[derive(Serialize, Deserialize, Debug)]
+#[serde(untagged)]
```

Comment:
This feature allows for untagged variant types, in this case a boolean or an object.